db/seeds.rb

```diff
@@ -1,7 +1,24 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+Instrument.destroy_all
+Booking.destroy_all
+User.destroy_all
+
+User.create!([
+  {first_name: Arisa, last_name: Nemoto, email: "ardolce23@xxx.com", password: "a23123123"},
+  {first_name: Alisha, last_name: Nemoto, email: "farbenmusik23@xxx.com", password: "1234567"}
+  ])
+
+10.times do 
+  Instrument.create!(
+    brand: Faker::Music.instrument,
+    location: ['Tokyo', 'Kyoto', 'Osaka', 'Hiroshima', 'Hokkaido'].sample,
+    renting_price: rand(10000..50000),
+    user_id: User.all.sample.id
+  )
```

Comment:
```
  instrument = Instrument.new(
    category: Instrument.categories.keys.sample,
    brand: ???,
    location: ['Tokyo', 'Kyoto', 'Osaka', 'Hiroshima', 'Hokkaido'].sample,
    renting_price: rand(10000..50000),
    condition: Instrument.conditions.keys.sample
  )
  instrument.user = User.all.sample
  instrument.save!
```

Reply:
➜  tunely git:(new-seeds) ✗ rails db:seed
Running via Spring preloader in process 51086
rake aborted!
ArgumentError: You tried to define an enum named "condition" on the model "Instrument", but this will generate a class method "new", which is already defined by Active Record.